portal/templates/redesign/help-and-support_new.html

```diff
@@ -0,0 +1,87 @@
+    <p class="body-text">The Code for Life website and the Rapid Router app should be supported by all modern browsers. Please note this means that Internet Explorer 8
+        and Internet Explorer 9 are not supported. Please upgrade to Internet Explorer 10 or above, or use an alternative browser if you are currently using these
+        browsers.</p><br>
+    <p class="body-text">Should you experience any difficulties with support for browsers or other devices you are using which you are unable to resolve, please
+        <a href="#contact">contact us</a> for assistance.</p>
+</div>
+<div id="faqs">
+    <div class="background col-sm-8 col-center">
+        <h3>FAQs</h3><br>
+        <p class="body-text"><b>+ I am a school student and have forgotten my password. How do I reset it?</b></p>
+        <p class="body-text"><b>+ I am a teacher and have forgotten my password. How do I reset it?</b></p>
+        <p class="body-text"><b>+ I am an independent student and have forgotten my password. How do I reset it?</b></p>
+        <p class="body-text"><b>+ I'm having problems choosing an acceptable password for my account. What am I missing?</b></p>
+        <p class="body-text"><b>+ How do I choose a secure and memorable password for my account?</b></p>
+        <p class="body-text"><b>+ What is two factor authentication? Do I need it?</b></p>
+        <p class="body-text"><b>+ What if I teach at more than one school? Can I sign up with one email address?</b></p>
+        <p class="body-text"><b>+ I want to sign up as a Teacher. Why am I asked for my 'professional' email address when registering?</b></p>
+        <p class="body-text"><b>+ What should I do when my students leave my school or club?</b></p>
+        <p class="body-text"><b>+ What is an independent student?</b></p>
+        <p class="body-text"><b>+ What if a student moves to another school? Can they keep their progress?</b></p>
+        <p class="body-text"><b>+ What do I need to do to allow an independent student to join my class or school?</b></p>
+        <p class="body-text"><b>+ I've set up my students. How do they log in?</b></p>
+        <p class="body-text"><b>+ As a student in a school or club, who can I share my custom created levels with?</b></p>
+        <p class="body-text"><b>+ As a teacher, who can I share my custom created levels with?</b></p>
+        <p class="body-text"><b>+ How can I control the levels my students create?</b></p>
+        <p class="body-text"><b>+ I am an independent student. Why can't I share my custom created levels?</b></p>
+    </div>
+</div>
+
+<div id="contact">
+    <div class="background background--primary">
+        <div class="col-sm-8 col-center">
+            <h3>Contact us</h3>
+            <form>
+                <div class="row">
+                    <div class="col-sm-6">
+                        <p class="small-text">Your name</p>
+                        <input type="text" placeholder="Firstname Lastname">
```

Comment:
Make sure you have a task for it in your backlog, or that one of your existing tasks contains it in the acceptance criteria.